modules/github.js

```diff
@@ -56,6 +60,50 @@ const github = {
     // #region Public Methods
     // -----------------------------------------------------------------------------------------
 
+    /**
+     * Adds a topic to all AndcultureCode repositories
+     *
+     * @param {string} topic Topic to be added
+     */
+    async addTopicToAllRepositories(topic) {
+        const andcultureRepos = await this.repositoriesByAndculture();
```

Comment:
Consider bailing on the method if the supplied topic is null or an empty string. I think you did something similar in the `addTopicToRepository` call using a call to `_validateTopicInputOrExit`.